### PR TITLE
Aida 410

### PR DESCRIPTION
**Pull request by gitnordt**

### CODE REVIEW

File: src/app/components/map/map.component.ts [hunk 1]
```diff
@@ -323,6 +331,51 @@ export class MapComponent extends BaseNeonComponent implements OnInit, OnDestroy
...
+                fieldName = field.columnName;
+            for (let filter of fieldFilters) {
+                fieldValues.push(filter.get(fieldName));
+            }
+            fieldValues = neonUtilities.flatten(fieldValues);
+            let fieldObject = this.createFilter(fieldsByLayer, ' ' + fieldName + ' = ' + fieldValues.join(',')),
+                neonIdFilter = this.createNeonFieldFilter(fieldName, fieldValues);
+            if (neonFilters && neonFilters.length) {
+                if (fieldObject) {
+                    this.replaceNeonFilter(this.options.layers[0], true, fieldObject, neonIdFilter);
```
Comment (ThomasSchellenbergNextCentury) [inline src/app/components/map/map.component.ts]: We should call manageFieldFilters on each layer in `options.layers`, and the layer (used on lines 371 and 374) and `fieldsByLayer` (line 360) should be passed-in as a function argument (don't rely on index 0).

**Review by ThomasSchellenbergNextCentury — CHANGES_REQUESTED** [review]

File: src/app/components/map/map.component.ts (hunk 1 above)
Comment (gitnordt) [reply src/app/components/map/map.component.ts]: I added the fix for this. I hope it is what you are looking for. If not, I may need some more guidance. Thanks!

Comment (ThomasSchellenbergNextCentury) [reply src/app/components/map/map.component.ts]: Looks good to me!

**Comment by ThomasSchellenbergNextCentury** [comment]
FYI I merged master into the branch to resolve merge conflicts.

**Review by ThomasSchellenbergNextCentury — APPROVED** [review]

**Review by bdpham — APPROVED** [review]

**Pull request merged**